cypress/utils/plugins/security-analytics-dashboards-plugin/commands.js

```diff
@@ -3,17 +3,478 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-require('./detectors');
-require('./rules');
-require('./typings');
+const {
+  OPENSEARCH_DASHBOARDS_URL,
+  OPENSEARCH_DASHBOARDS,
+} = require('./constants');
+const { NODE_API } = require('./constants');
+const { BACKEND_BASE_PATH } = require('../../base_constants');
+
+// ***********************************************
+// This example commands.js shows you how to
+// create various custom commands and overwrite
+// existing commands.
+//
+// For more comprehensive examples of custom
+// commands please read more here:
+// https://on.cypress.io/custom-commands
+// ***********************************************
+//
+//
+// -- This is a parent command --
+// Cypress.Commands.add("login", (email, password) => { ... })
+//
+//
+// -- This is a child command --
+// Cypress.Commands.add("drag", { prevSubject: 'element'}, (subject, options) => { ... })
+//
+//
+// -- This is a dual command --
+// Cypress.Commands.add("dismiss", { prevSubject: 'optional'}, (subject, options) => { ... })
+//
+//
+// -- This will overwrite an existing command --
+// Cypress.Commands.overwrite("visit", (originalFn, url, options) => { ... })
 
 Cypress.Commands.add('cleanUpTests', () => {
```

Comment:
Not for this PR but we should namespace this too.

Reply:
Done